recipes/mapseq/meta.yaml

```diff
@@ -1,16 +1,18 @@
-{% set version = "2.1.1" %}
-{% set sha256 = "b66af114f82ed9671e250995d014704b58b8e19b7ec95b1ef05a02eb8c5e8018" %}
+{% set version = "2.1.1a" %}
+{% set sha256 = "af0cb71824dd5a9b31a54b486e8444d0fb6929b4b96723f81b8a33b639aa21f7" %}
 
 package:
   name: mapseq
   version: '{{ version }}'
 
 source:
-  url: https://github.com/jfmrod/MAPseq/archive/v{{ version }}.tar.gz
+  url: https://github.com/meringlab/MAPseq/archive/v{{ version }}.tar.gz
```

Comment:
@jfmrod is that ok for you?

Reply:
Might be preferable to make this a fork (currently it seems to be just a clone pushed to a new repo)?  Than the connection between the two repos would be clearer.